app/assets/javascripts/discourse/app/controllers/reject-reason-reviewable.js

```diff
@@ -0,0 +1,16 @@
+import Controller from "@ember/controller";
+import ModalFunctionality from "discourse/mixins/modal-functionality";
+
+export default Controller.extend(ModalFunctionality, {
+  onShow() {
```

Comment:
It is good practice to declare local variables when defining your class. Javascript is better at keeping the same object shape rather than changing it later.

example:
```
  rejectReason: null,
  sendEmail: false
```